source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -268,17 +285,75 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         }
     }
 
+    winrt::fire_and_forget ConfigureSvgImageSourceAsync(winrt::Uri uri, winrt::SvgImageSource svgImageSource, winrt::TileControl tileControl)
+    {
+        auto weakImageSource{winrt::make_weak(svgImageSource)};
+
```

Comment:
can you take weakRef of tileControl? and check if it's still alive when it get used?